dhis-2/dhis-web-api/src/main/resources/openapi/EnrollmentsExportController.md

```diff
@@ -53,73 +55,75 @@ excluded by default.
 
 `<orgUnit1-uid>[,<orgUnit2-uid>...]`
 
-Get enrollments owned by given `orgUnits` relative to the `orgUnitMode`.
+Get enrollments owned by given organisation units in `orgUnits` parameter relative to the 
+`orgUnitMode` parameter.
 
 - When `orgUnitMode=SELECTED` - or no `orgUnitMode` is given (default) - the enrollments owned by
   the `orgUnits` are returned.
 - When `orgUnitMode=CHILDREN` the enrollments owned by the `orgUnits` or by the `orgUnits` direct
-  children is returned.
+  children are returned.
 - When `orgUnitMode=DESCENDANTS` the enrollments owned by the `orgUnits` or any of its descendants
   are returned.
 - When `orgUnitMode=ALL`, `orgUnitMode=CAPTURE` or `orgUnitMode=ACCESSIBLE` the `orgUnits` parameter
   is not allowed.
 
 ### `*.parameter.EnrollmentRequestParams.orgUnitMode`
 
-Get enrollments using given organisation unit selection mode.
+Get enrollments using the given organisation unit selection mode.
 
-- When `orgUnitMode=SELECTED`, `orgUnitMode=CHILDREN` or `orgUnitMode=DESCENDANTS`, the `orgUnit`
-  parameter is
-  required to specify which enrollments to return.
+- When `orgUnitMode=SELECTED`, `orgUnitMode=CHILDREN` or `orgUnitMode=DESCENDANTS`, the `orgUnits`
+  parameter is required to specify which enrollments to return.
 - When `orgUnitMode=ALL` enrollments will be downloaded irrespective of the organization unit they
```

Comment:
```suggestion
- When `orgUnitMode=ALL` enrollments will be returned irrespective of the organization unit they
```

since we talk about returning enrollments in the entire doc